test/table.js

```diff
@@ -174,6 +175,20 @@ describe('Table', () => {
       assert.deepEqual(rows.length, 231)
     })
 
+    it('should autodetect CSV separated by semicolons', async () => {
+      const data = `city;location;population
```

Comment:
Use `data_semicolon_separated.csv`?

Reply:
Oh, I need to remove the file. I added it directly here so it tests both the browser and node.